src/ec/packages/ec-component-description-list/description-list.story.js

```diff
@@ -22,20 +26,35 @@ const prepareList = data => {
   }
   data.items.forEach((item, i) => {
     if (Array.isArray(item.term)) {
-      item.term.forEach((t, ind) => {
-        t = text(`items[${i}].term[${ind}]`, t, tabLabels.required);
+      item.term.forEach((term, ind) => {
+        data.items[i].term[ind] = text(
+          `items[${i}].term[${ind}]`,
+          term,
+          tabLabels.required
+        );
       });
     } else {
       item.term = text(`items[${i}].term`, item.term, tabLabels.required);
     }
-    item.definition = text(
-      `items[${i}].definition`,
-      item.definition,
-      tabLabels.required
-    );
+    if (Array.isArray(item.definition)) {
+      item.definition.forEach((definition, ind) => {
+        data.items[i].definition[ind] = text(
+          `items[${i}].definition[${ind}]`,
+          definition,
+          tabLabels.required
+        );
+      });
+    } else if (item.definition) {
```

Comment:
Modifying single-value definition items inside the knobs UI make them disappear similarly to many other items commented in previous pull requests.
Not sure whether if/else without a default case is not causing this.